playground/steps/simplePython/testEvent.step.py

```diff
@@ -0,0 +1,21 @@
+config = {
+    "type": "event",
+    "name": "Test Event",
+    "subscribes": ["test"],
+    "emits": ["tested"],
+    "input": None,  # No schema validation in Python version
```

Comment:
We could also add input validation here and a test that passes and another that fails.